### PR TITLE
[no ticket][risk=no] Tweaks to PR checklist

### PR DESCRIPTION
**Pull request by dmohs**
PR checklist tweaks.

---
**PR checklist**

- [X] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [X] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [x] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [X] None of the above apply to this change.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-**Description**
-
```
Comment (dmohs) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: I don't think this needs a placeholder as it is implied.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 <!--
 Reminder: If you decide to merge with any failing checks, add an explanatory comment before doing so.
 -->
 
 ---
 **PR checklist**
 
-- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
```
Comment (dmohs) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: I think these are all sentences, so I have added punctuation.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
-- [ ] If this includes an API change, I have run the relevant E2E tests locally because API changes are not covered by our PR checks
+- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
+- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
+- [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
+- [ ] I have manually tested this change and my testing process is described above.
+- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
+- [ ] I have added explanatory comments where the logic is not obvious.
+- One more of the following is true:
```
Comment (jmthibault79) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: "one or more"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-- [ ] I have added explanatory comments where the logic is not obvious
-- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented the impacts in the description
-- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
-- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
-- [ ] If this includes an API change, I have run the relevant E2E tests locally because API changes are not covered by our PR checks
+- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
+- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
+- [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
```
Comment (jmthibault79) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: This should be moved to the "one or more" section, because it doesn't apply to all PRs.

**Review by jmthibault79 — APPROVED** [review]
I like the wording changes and the split into mandatory vs. optional sections

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (dmohs) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks for catching that.

**Pull request merged**